hspec-core/src/Test/Hspec/Core/Formatters/Pretty/Parser.hs

```diff
@@ -16,12 +16,6 @@ import           Prelude ()
 import           Test.Hspec.Core.Compat hiding (fail)
 import           Test.Hspec.Core.Formatters.Pretty.Parser.Types
 
-#ifndef __GHCJS__
```

Comment:
I think we want to keep this check (see #599).

Reply:
Actually I'm not sure this is the case (and CI doesn't seem to include GHCJS), so I'll include this condition just to be safe.